### ISSUE TITLE
Found error causing bag not to open with certain addons.

### ISSUE THREAD

**Issue by Vashtstamped** [tullamods/Bagnon#754]
Error is located at Bagnon\external\Wildpants\libs\LibItemCache-2.0

In > function Lib:IterateOwners()

Was working fine in ver 7.3.8 and was broken after changes in 7.3.9 and above.  Using the old function Lib:IterateOwners() from 7.3.8 works in the newer versions in my testing.

Fixes bags not opening when autobar, zygor, and skillet are all used.

**Comment by Vashtstamped** [comment]
[LibItemCache-2.0.lua.zip](https://github.com/tullamods/Bagnon/files/2279951/LibItemCache-2.0.lua.zip)

I updated the file with the Lib:IterateOwners() changes and uploaded for your testing.  Seems to be working fine now for me on Zul'gin servers.

**Comment by Braunro** [comment]
This did nothing to fix the problem for me. I use Autobar and Skillet but not Zygor. Bagnon version 8.0.0b works for me but not the later versions. 8.0.0b uses LibItemCache-1.1

**Comment by Vashtstamped** [comment]
Did you update to 8.0.2 and then try adding the file to Bagnon\external\Wildpants\libs\ .

Also make sure wow is closed as it will hold the old file in memory.  I just disabled zygor and running autobar and skillet with no problems.

**Comment by Braunro** [comment]
Yes, I used 8.0.2. I even tried completely deleting Bagnon before installing it. I even tried installing it, starting WoW and "opening" my bags several times, then closing WoW and copying your lib file over before starting WoW and trying again. Yes, I closed WoW in between all attempts when trying all of this. I spent about 2 and a 1/2 hours trying various things including deleting my Bagnon variables.

**Comment by Braunro** [comment]
This issue has been resolved for me. Autobar and Skillet made updates to work around where Bagnon was conflicting. That fixed it for me and it stayed fixed after the latest Bagnon update.
